trex.org

```diff
@@ -272,41 +326,66 @@
 *** Data definitions
 
     #+NAME: basis
-    | Variable        | Type    | Dimensions          | Description                                                     |
-    |-----------------+---------+---------------------+-----------------------------------------------------------------|
-    | ~type~          | ~str~   |                     | Type of basis set: "Gaussian", "Slater" or "PW" for plane waves |
-    | ~prim_num~      | ~dim~   |                     | Total number of primitives                                      |
-    | ~shell_num~     | ~dim~   |                     | Total number of shells                                          |
-    | ~nucleus_index~ | ~index~ | ~(basis.shell_num)~ | One-to-one correspondence between shells and atomic indices     |
-    | ~shell_ang_mom~ | ~int~   | ~(basis.shell_num)~ | One-to-one correspondence between shells and angular momenta    |
-    | ~shell_factor~  | ~float~ | ~(basis.shell_num)~ | Normalization factor of each shell ($\mathcal{N}_s$)            |
-    | ~r_power~       | ~int~   | ~(basis.shell_num)~ | Power to which $r$ is raised ($n_s$)                            |
-    | ~shell_index~   | ~index~ | ~(basis.prim_num)~  | One-to-one correspondence between primitives and shell index    |
-    | ~exponent~      | ~float~ | ~(basis.prim_num)~  | Exponents of the primitives ($\gamma_{ks}$)                     |
-    | ~coefficient~   | ~float~ | ~(basis.prim_num)~  | Coefficients of the primitives ($a_{ks}$)                       |
-    | ~prim_factor~   | ~float~ | ~(basis.prim_num)~  | Normalization coefficients for the primitives ($f_{ks}$)        |
-    | ~e_cut~         | ~float~ |                     | Energy cut-off for plane-wave calculations                      |
+    | Variable            | Type    | Dimensions                                   | Description                                                                  |   |
+    |---------------------+---------+----------------------------------------------+------------------------------------------------------------------------------+---|
+    | ~type~              | ~str~   |                                              | Type of basis set: "Gaussian", "Slater", "Numerical" or "PW" for plane waves |   |
+    | ~prim_num~          | ~dim~   |                                              | Total number of primitives                                                   |   |
+    | ~shell_num~         | ~dim~   |                                              | Total number of shells                                                       |   |
+    | ~numgrid_num~       | ~dim~   |                                              | Total number of grid points for numerical orbitals                           |   |
```

Comment:
Maybe `grid_point` instead of `numgrid`?

Reply:
And general question: why not re-using an existing `grid` group? We have `num` parameter there.

Reply:
> And general question: why not re-using an existing grid group? We have num parameter there.

Here, each AO has its own grid so it is probably safer not to mix the global integration grid with the grids of the AOs.

I also find the name `numgrid` confusing, because there is a package called `numgrid` to generate grids for DFT calculations. Users could be mislead by this name. We could simply use `grid` or `grid_point` as suggested by @q-posev